src/prerecorded/options/mod.rs

```diff
@@ -0,0 +1,337 @@
+                Language::en_NZ => "en-NZ",
+                Language::fr => "fr",
+                Language::fr_CA => "fr-CA",
+                Language::de => "de",
+                Language::hi => "hi",
+                Language::id => "id",
+                Language::it => "it",
+                Language::ja => "ja",
+                Language::ko => "ko",
+                Language::pt => "pt",
+                Language::pr_BR => "pr_BR",
+                Language::ru => "ru",
+                Language::es => "es",
+                Language::es_419 => "es-419",
+                Language::sv => "sv",
+                Language::tr => "tr",
+                Language::uk => "uk",
+            };
+
+            seq.serialize_element(&("language", s))?;
+        }
+
+        if let Some(punctuate) = punctuate {
+            seq.serialize_element(&("punctuate", punctuate))?;
+        }
+
+        if let Some(profanity_filter) = profanity_filter {
+            seq.serialize_element(&("profanity_filter", profanity_filter))?;
+        }
+
+        for element in redact {
+            let s = match element {
+                Redact::Pci => "pci",
+                Redact::Numbers => "numbers",
+                Redact::Ssn => "ssn",
+            };
+
+            seq.serialize_element(&("redact", s))?;
+        }
+
+        if let Some(diarize) = diarize {
+            seq.serialize_element(&("diarize", diarize))?;
+        }
+
+        if let Some(ner) = ner {
+            seq.serialize_element(&("ner", ner))?;
+        }
+
+        if let Some(multichannel) = multichannel {
+            seq.serialize_element(&("multichannel", multichannel))?;
+        }
+
+        if let Some(alternatives) = alternatives {
+            seq.serialize_element(&("alternatives", alternatives))?;
+        }
+
+        if let Some(numerals) = numerals {
+            seq.serialize_element(&("numerals", numerals))?;
+        }
+
+        for element in search {
+            seq.serialize_element(&("search", element))?;
+        }
+
+        if let Some(callback) = callback {
+            seq.serialize_element(&("callback", callback))?;
+        }
+
+        for element in keywords {
+            seq.serialize_element(&("keywords", element))?;
+        }
+
+        match utterances {
+            Some(Utterances::Disabled) => seq.serialize_element(&("utterances", false))?,
+            Some(Utterances::Enabled { utt_split: None }) => {
+                seq.serialize_element(&("utterances", true))?
+            }
+            Some(Utterances::Enabled {
+                utt_split: Some(utt_split),
+            }) => {
+                seq.serialize_element(&("utterances", true))?;
+                seq.serialize_element(&("utt_split", utt_split))?;
+            }
+            None => (),
+        };
+
+        if let Some(tag) = tag {
+            seq.serialize_element(&("tag", tag))?;
+        }
+
+        seq.end()
+    }
+}
+
+#[cfg(test)]
+mod serialize_options_tests;
```

Comment:
Normally, we keep unit test modules in-file with the module they test.
```rust
#[cfg(test)]
mod tests {
   ...
}
```

Reply:
Implemented in 63787b227a93fac0951f60dfdfe03c1bc81086b3.